docs/notices_and_disclaimers.rst

```diff
@@ -0,0 +1,16 @@
+.. # Copyright (C) 2020-2021 Intel Corporation
+.. # SPDX-License-Identifier: Apache-2.0
+
+***********************
+Notices and Disclaimers
+***********************
+
+© Intel Corporation.  Intel, the Intel logo, and other Intel marks are trademarks of Intel Corporation or its subsidiaries.  Other names and brands may be claimed as the property of others. 
```

Comment:
Replace `.` with `,` . `© Intel Corporation, Intel, the Intel logo,...`

Reply:
This statement was copied from legal. I prefer not to change how the statement is presented. However, I've removed the double spaces in the sentence.